docs/KNOWLEDGE-WORKERS.md

```
@@ -61,12 +61,12 @@ graph TD
     E --> F[UX Insights & Prioritized Backlog]
 ```
 
-#### Phase 1: Research Synthesis (Gemini)
+#### Phase 1: Research Synthesis (Antigravity)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)